arches/app/tasks.py

```diff
@@ -74,6 +77,7 @@ def export_search_results(self, userid, request_values, format):
         closing="Thank you",
         email=email,
         name=export_name,
+        email_link=str(settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT) + "/files/" + str(search_history_obj.downloadfile),
```

Comment:
I think the `settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT` has a trailing forward slash - at least the settings file says it should. Therefore, I would suggest that a right strip be used to remove it if it is there...

```suggestion
        email_link=str(settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT).rstrip("/") + "/files/" + str(search_history_obj.downloadfile),
```

Reply:
That should be in the second commit now.